README.md

```diff
@@ -1,2 +1,57 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Wamae-Portfolio
-Portfolio Website To Showcase Projects
+
+> Portfolio Project To Showcase Projects
+
+
+## Built With
+
+- Html
+- Css
+
+
+
+## Getting Started
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+ - Browser
+ - Git
+### Setup
+	Clone this repository
+### Install
+	Open Html file in preferred browser
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@mosdevx](https://github.com/mosdevx)
+- Twitter: [@moseswamae7](https://twitter.com/moseswamae7)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/moses-wamae-a13a67244)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] You have a link here to your LICENSE file :+1: but your forgot to add the LICENSE file. :smile: Kindly add the license file using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :+1:

Reply:
Thanks. Ive rectified both issues